server/routes/index.js

```diff
@@ -26,6 +32,7 @@ router.post('/type', isAuth, isAdmin, postType);
 router.get('/brands', getBrands);
 router.get('/logout', logout);
 router.delete('/admin/deleteCar/carId', isAuth, isAdmin, deleteCar);
+router.delete('/delete/type/:typeId', isAuth, isAdmin, deleteTypeControllers);
```

Comment:
router.delete('/type/:typeId', isAuth, isAdmin, deleteTypeControllers);

Reply:
Yes and it's better to be:
```suggestion
router.delete('/admin/type/:typeId', isAuth, isAdmin, deleteTypeControllers);
```